model/cluster_utility.go

```diff
@@ -64,7 +64,7 @@ func GetGitlabToken() string {
 // DefaultUtilityVersions holds the default values for all the HelmUtilityVersions
 var DefaultUtilityVersions map[string]*HelmUtilityVersion = map[string]*HelmUtilityVersion{
 	// PrometheusOperatorCanonicalName defines the default version and values path for the Helm chart
-	PrometheusOperatorCanonicalName: {Chart: "48.1.2", ValuesPath: ""},
+	PrometheusOperatorCanonicalName: {Chart: "54.2.2", ValuesPath: ""},
```

Comment:
Just checking that this chart version correlates to `v0.69.1` of the prometheus operator.

Reply:
I believe it's https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/Chart.yaml#L25

Reply:
Yeap this is the chart for the Operator version :)